### PR TITLE
st2 [1820][IMP] stock_view_adj_hls: adjust lot_name

### PR DESCRIPTION
**Pull request by TomohiroAbe-Quartile**
[1820](https://www.quartile.co/web#id=1820&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: stock_view_adj_hls/views/stock_quant_views.xml [hunk 1]
```diff
@@ -25,4 +25,16 @@
             </xpath>
         </field>
     </record>
+
+    <record id="view_stock_move_line_operation_tree" model="ir.ui.view">
```
Comment (yostashiro) [inline stock_view_adj_hls/views/stock_quant_views.xml]: Please create a new file `stock_move_line_views.xml` and put this record in there.

Ref. https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming

File: stock_view_adj_hls/views/stock_quant_views.xml [hunk 2]
```diff
@@ -25,4 +25,16 @@
...
+        <field name="name">stock.move.line.operations.tree</field>
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id"
+               ref="stock.view_stock_move_line_operation_tree"/>
+        <field name="arch" type="xml">
+            <field name="lot_id" position="attributes">
+                <attribute name="readonly">1</attribute>
```
Comment (yostashiro) [inline stock_view_adj_hls/views/stock_quant_views.xml]: Any reason this should be readonly instead of invisible?

**Review by yostashiro — CHANGES_REQUESTED** [review]
Please update DESCRIPTION.rst to record the intentions and the reasons of this change.

File: stock_view_adj_hls/views/stock_quant_views.xml [hunk 3]
```diff
@@ -25,4 +25,16 @@
...
         </field>
     </record>
+
+    <record id="view_stock_move_line_operation_tree" model="ir.ui.view">
+        <field name="name">stock.move.line.operations.tree</field>
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id"
+               ref="stock.view_stock_move_line_operation_tree"/>
+        <field name="arch" type="xml">
+            <field name="lot_id" position="attributes">
```
Comment (yostashiro) [inline stock_view_adj_hls/views/stock_quant_views.xml]: @TomohiroAbe-Quartile Actually, why `lot_id`? Shouldn't we be adjusting `lot_name`?

File: stock_view_adj_hls/views/stock_quant_views.xml (hunk 2 above)
Comment (TomohiroAbe-Quartile) [reply stock_view_adj_hls/views/stock_quant_views.xml]: @yostashiro 
I think there is a situation where want to check the lot_id, so I set it to readonly.

File: stock_view_adj_hls/views/stock_quant_views.xml (hunk 3 above)
Comment (TomohiroAbe-Quartile) [reply stock_view_adj_hls/views/stock_quant_views.xml]: @yostashiro 
I edit this field.

Comment (TomohiroAbe-Quartile) [reply stock_view_adj_hls/views/stock_quant_views.xml]: I confirmed lot_name field and add invisible attribute

File: stock_view_adj_hls/views/stock_quant_views.xml (hunk 1 above)
Comment (TomohiroAbe-Quartile) [reply stock_view_adj_hls/views/stock_quant_views.xml]: Add new file.

And edit [description.rst.](https://github.com/qrtl/hls-custom/pull/141/files#diff-856934fdd42372f4d9ada7c9abdaa7fdec5fd6846d2ce5e334be3f246e7b810bR1)

File: stock_view_adj_hls/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1 @@
+This module extends field in stock_quant_view and following stock_move_line view.
```
Comment (yostashiro) [inline stock_view_adj_hls/readme/DESCRIPTION.rst]:
```suggestion
This module makes adjustments to some of the stock related views.

Some points to note:

* `lot_name` in stock move line operation tree is made invisible in view of using
  the OCA module stock_picking_auto_create_lot - the users will not be manually
  keying in the lot number during the receipt operation.
```

Please re-generate README.rst.

**Review by yostashiro — CHANGES_REQUESTED** [review]
Please bump the module version (patch) as well.

File: stock_view_adj_hls/readme/DESCRIPTION.rst (hunk 4 above)
Comment (TomohiroAbe-Quartile) [reply stock_view_adj_hls/readme/DESCRIPTION.rst]: I edited DESCRIPTION and re-generate README.rst

File: stock_view_adj_hls/README.rst [hunk 5]
```diff
@@ -28,10 +36,10 @@ Stock View Adj HLS
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues <https://github.com/qrtl/hls-custom/issues>`_.
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/hls-custom/issues>`_.
```
Comment (yostashiro) [inline stock_view_adj_hls/README.rst]: @TomohiroAbe-Quartile Please regenerate README.rst with `qrtl` organization.

Comment (TomohiroAbe-Quartile) [reply stock_view_adj_hls/README.rst]: Sorry, now regenerate README with qrtl organizetion.

File: stock_view_adj_hls/static/description/index.html [hunk 6]
```diff
@@ -381,10 +386,10 @@ <h1 class="title">Stock View Adj HLS</h1>
 </div>
 <div class="section" id="bug-tracker">
 <h1><a class="toc-backref" href="#id1">Bug Tracker</a></h1>
-<p>Bugs are tracked on <a class="reference external" href="https://github.com/qrtl/hls-custom/issues">GitHub Issues</a>.
+<p>Bugs are tracked on <a class="reference external" href="https://github.com/OCA/hls-custom/issues">GitHub Issues</a>.
```
Comment (yostashiro) [inline stock_view_adj_hls/static/description/index.html]: Why is the organization not updated here?  The same applies to the other changed parts in this file.

File: stock_view_adj_hls/README.rst [hunk 7]
```diff
@@ -31,7 +33,7 @@ Bug Tracker
 Bugs are tracked on `GitHub Issues <https://github.com/qrtl/hls-custom/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us smashing it by providing a detailed and welcomed
-`feedback <https://github.com/qrtl/hls-custom/issues/new?body=module:%20stock_view_adj_hls%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+`feedback <https://github.com/qrtl/hls-custom/issues/new?body=module:%20stock_view_adj_hls%0Aversion:%201820-make-serial-number-uneditable%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (yostashiro) [inline stock_view_adj_hls/README.rst]: Please use the 12.0 branch when generating the readme file.

**Review by yostashiro — CHANGES_REQUESTED** [review]

File: stock_view_adj_hls/static/description/index.html (hunk 6 above)
Comment (TomohiroAbe-Quartile) [reply stock_view_adj_hls/static/description/index.html]: Sorry, I misunderstand how to generate.
Now re-generate index.html, confirmed changed

**Review by yostashiro — APPROVED** [review]
LGTM 👍

**Pull request merged**